### PR TITLE
Fix #7 Add tailwind css to project

### LINKED ISSUE #7: Add tailwind css to project
Add tailwind css to project

### PR DESCRIPTION
**Pull request by rray524**
Fix #7 Add tailwind css to project

### Description

This pull request adds detailed instructions on how to set up and use Tailwind CSS in a React project. This PR also includes the necessary npm package for Tailwind CSS and updates the configuration files.

### Type of Change

* Added Tailwind CSS npm package to the project.
* Configuration details for setting up Tailwind CSS in the project.

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/** @type {import('tailwindcss').Config} */
+export const content = ["./src/**/*.{js,jsx,ts,tsx}"];
```
Comment (rt4914) [inline tailwind.config.js:2]: Include `"./index.html",` too in this.

**Review by rt4914 — CHANGES_REQUESTED** [review]
One minor change

File: tailwind.config.js (hunk 1 above)
Comment (rray524) [reply tailwind.config.js]: @rt4914 added. thanks.

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

**Pull request merged**